store/src/java/com/zimbra/cs/mailbox/Folder.java

```diff
@@ -1691,7 +1691,7 @@ public boolean isIMAPSubscribed() {
      **/
     @Override
     public int getUIDValidity() {
-        return Math.max(getSavedSequence(), 1);
+        return (int)Math.max(getSavedSequenceLong(), 1);
```

Comment:
I guess this is no worse than the cast below it, but what is the expected behavior if the sequence is bigger than int max here?

Reply:
This we can change with the hope that its not used anywhere else except internal IMAP code. changing into to long is no harm if its method argument, but if its return type, then it can break the things.